projects/assets-library/assets/icons/minus-circle-reverse.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M5.8335 10C5.8335 10.4602 6.20659 10.8333 6.66683 10.8333H13.3335C13.7937 10.8333 14.1668 10.4602 14.1668 10C14.1668 9.53976 13.7937 9.16667 13.3335 9.16667H6.66683C6.20659 9.16667 5.8335 9.53976 5.8335 10Z" fill="white"/>
```

Comment:
```suggestion
<path d="M5.8335 10C5.8335 10.4602 6.20659 10.8333 6.66683 10.8333H13.3335C13.7937 10.8333 14.1668 10.4602 14.1668 10C14.1668 9.53976 13.7937 9.16667 13.3335 9.16667H6.66683C6.20659 9.16667 5.8335 9.53976 5.8335 10Z" fill="currentColor"/>
```

Svg fills should be currentColor so we can style them externally

Reply:
done